src/js/video.js

```diff
@@ -397,6 +398,20 @@ videojs.log = log;
 videojs.createTimeRange = createTimeRange;
 
 /**
+ * @file format-time.js
```

Comment:
Ah, I see. copy/paste. The `@file` isn't needed here, and could probably be moved into it's own comment block in format-time.js to avoid future issues like this.